lib/dry/cli/command.rb

```diff
@@ -50,7 +50,7 @@ def self.extended(base)
         # @api private
         attr_reader :options
 
-        # @since 0.6.x
+        # @since 0.7.0
```

Comment:
Is this and all the other `# @since` changes meant to be included? They seem unrelated to the rest of the PR...

Reply:
this is because it was submitted before in another PR

Reply:
Sorry, I still don’t understand... this PR isn’t bumping a version, and it’s not altering these methods... so why are we modifying their doc comments?

Reply:
Because the previous PR with these changes was merged already, it contained 0.6.x comments. `0.6.x` is easy to find in a project to bump all docs simultaneously at the moment of new release.  
@jodosha asked to fix `API docs report @since 0.6.x, but they should be @since 0.7.0.` and I just did it. These methods will first appear in 0.7.0. So this comment is 100% valid to have.

Reply:
OK, makes sense, thanks for explaining!